crypto/evp/evp_lib.c

```diff
@@ -368,10 +368,8 @@ int evp_cipher_cache_constants(EVP_CIPHER *cipher)
             cipher->flags |= EVP_CIPH_FLAG_CTS;
         if (multiblock)
             cipher->flags |= EVP_CIPH_FLAG_TLS1_1_MULTIBLOCK;
-        /* Provided implementations may have a custom cipher_cipher */
-        if (cipher->prov != NULL && cipher->ccipher != NULL)
```

Comment:
I am not entirely sure I understand the path for coverity to complain about.
I see that we do check `cipher->prov` against NULL in `evp_do_ciph_getparams()`, but I am not sure where it is being dereferenced.

Reply:
Inside the later call to `EVP_CIPHER_gettable_ctx_params()`